backend/app/db/mysql_loop_items_schema.py

```
@@ -0,0 +1,399 @@
+        "delivery_id",
+        "public_id",
+        "project_key",
+        "storage_prefix",
+    )
+    for name in string_columns:
+        column = _quote(connection, name)
+        assignments.append(f"{column} = NULLIF({column}, '')")
+    for name in ("local_project_id", "backend_task_id"):
+        column = _quote(connection, name)
+        assignments.append(f"{column} = NULLIF({column}, 0)")
+    connection.exec_driver_sql("UPDATE `loop_items` SET " + ", ".join(assignments))
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)